hm_1/tasks/task_01_03.py

```diff
@@ -0,0 +1,20 @@
+"""
+Write down the function, which reads input line-by-line, and find maximum and minimum values.
+Function should return a tuple with the max and min values.
+For example for [1, 2, 3, 4, 5], function should return [1, 5]
+We guarantee, that file exists and contains line-delimited integers.
+To read file line-by-line you can use this snippet:
+with open("some_file.txt") as fi:
+    for line in fi:
+        ...
+"""
+from typing import Tuple
+
+
+def find_maximum_and_minimum(file_name: str) -> Tuple[int, int]:
+    data = []
+    with open(file_name) as fi:
+        for line in fi:
+            data.append(int(line.rstrip("\n")))
+        data.sort()
```

Comment:
What is the complexity of this line?  Can we do better?

Reply:
I don't know how to do this O(1)

Reply:
> I think It's O(len(fi))

It is not an exam, where I check your knowledge, it is a learning process.  I expect that you check your answer in the documentation.

> I don't know how to do this O(1)

It is not possible, O(l) where l is the number of lines in files expected here.